config/locales/en.yml

```diff
@@ -156,6 +156,8 @@ en:
     issue_status: 'Status'
     issue_assignee: 'Assignee'
     not_assigned: '----'
+    issue_documents: 'Documents'
```

Comment:
Just 'document' as key name would be fine. No need to append issue.

Reply:
ok